### ISSUE TITLE
FR: Option to hide tab row

### ISSUE THREAD

**Issue by Gitoffthelawn** [WinMerge/winmerge#547]
WinMerge has a tab row located between the main toolbar and the actual comparison content.  I don't think I've ever used the tab row, as I typically use WinMerge to compare files (or folders), and then close it.

Could we have an option to not display that tab row?

I'm thinking these are the benefits:

1. Simplify the UI.  When the user is comparing 2 files (which is likely the most common use), it provides no useful info and just clutters up the UI.
2. Make the UI less confusing.  When the user is comparing 2 files, part of the tab label will contain the name of the file that is displayed on the other half of the screen, making it confusing as to which data is actually displayed below the tab label.
3. Allow for more content to be shown.  If the user could choose to hide the tab bar, there is more room for comparison data (the most important data).

**Comment by sdottaka** [comment]
If you uncheck View-> Tab bar menu item, Tab row (Tab bar) will be hidden.

![image](https://user-images.githubusercontent.com/98126/104113061-3efde500-5339-11eb-880a-dd76ecd5d284.png)

**Comment by Gitoffthelawn** [comment]
@sdottaka LOL.  Before posting, I went through all the pages in the settings UI thrice, and even went through every item in the settings export file.   Originally, I was just going to ask if this was already possible, but after doing all that checking, I figured it was safe to make the above proposal.  LOL.  I appreciate your response, and I must laugh at not seeing that toggle in the View menu.

I just verified that the Tab Bar toggle accomplishes exactly what I proposed, and it does maintain its setting between WinMerge launches.  WFM.

I'm going to close this now with my thanks, and if you see any need to reopen it to further address or discuss this functionality, feel free.
